game/lib/global_leaderboard/podium_widgets.dart

```diff
@@ -15,89 +23,63 @@ var pointsStyle = TextStyle(
   height: 0,
 );
 
-Widget FirstPodium(context, int points, bool isUser) {
-  // Choosing whetehr podum is highlighted or not by isUser condition
-  String svgAssetPath = isUser
-      ? 'assets/icons/podium1highlighted.svg'
-      : 'assets/icons/podium1red.svg';
-
-  return Center(
-      child: Container(
-          width: 106,
-          height: 112,
-          child:
-              Stack(alignment: FractionalOffset(0.5, 0.58), children: <Widget>[
-            SvgPicture.asset(
-              svgAssetPath,
-              semanticsLabel: '1st Podium',
-            ),
-
-            SizedBox(
-              width: 60,
-              height: 29,
-              child: FittedBox(
-                alignment: Alignment.center,
-                fit: BoxFit.scaleDown,
-                child: Text(points.toString() + " PTS", style: pointsStyle),
-              ),
-            ),
-            // Text(points.toString() + " PTS"
-          ])));
-}
-
-Widget SecondPodium(context, int points, bool isUser) {
-  String svgAssetPath = isUser
-      ? 'assets/icons/podium2highlighted.svg'
-      : 'assets/icons/podium2red.svg';
-
-  return Center(
-      child: Container(
-          width: 106,
-          height: 86,
-          child:
-              Stack(alignment: FractionalOffset(0.5, 0.63), children: <Widget>[
-            SvgPicture.asset(
-              svgAssetPath,
-              semanticsLabel: '2nd Podium',
-            ),
-
-            SizedBox(
-              width: 60,
-              height: 29,
-              child: FittedBox(
-                alignment: Alignment.center,
-                fit: BoxFit.scaleDown,
-                child: Text(points.toString() + " PTS", style: pointsStyle),
-              ),
-            ),
-            // Text(points.toString() + " PTS"
-          ])));
-}
+Widget PodiumBlock(context, int position, List scoreList) {
+  Map<int, String> svgAssetPaths = {
+    0: 'assets/icons/blank_podiums.svg',
+    1: 'assets/icons/gold_podiums.svg',
+    2: 'assets/icons/silver_podiums.svg',
+    3: 'assets/icons/bronze_podiums.svg',
+  };
 
-Widget ThirdPodium(context, int points, bool isUser) {
-  String svgAssetPath = isUser
-      ? 'assets/icons/podium3highlighted.svg'
-      : 'assets/icons/podium3red.svg';
-  return Center(
-      child: Container(
-          width: 106,
-          height: 62,
-          child:
-              Stack(alignment: FractionalOffset(0.5, 0.75), children: <Widget>[
-            SvgPicture.asset(
-              svgAssetPath,
-              semanticsLabel: '3rd Podium',
-            ),
```

Comment:
Your previous way of switching between highlighted and not podiums is a lot more efficient than this new mapping you have where you import svgs of all 3 podiums + trophies + labels. Is there a reason you couldn't add each individual trophy on top of your existing cylinders?